cmd/users.go

```diff
@@ -496,6 +496,25 @@ var RemoveUserFromOrganization = &cobra.Command{
 	},
 }
 
+var resetPasswordCmd = &cobra.Command{
+	Use:     "reset-password",
+	Aliases: []string{"reset-pass", "rp"},
+	Short:   "Send a password reset email",
+	PersistentPreRun: func(cmd *cobra.Command, args []string) {
+		validateToken(lagoonCLIConfig.Current) // get a new token if the current one is invalid
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		if userEmail == "" {
```

Comment:
I think it would be good to prompt for confirmation for this particular action, an example is here
https://github.com/uselagoon/lagoon-cli/blob/6662fa5af70186f1cee892b019775d03087de5e3/cmd/deploy.go#L59

Reply:
Discussed out of band, agree it fits in with the rest of the commands :+1: